src/graph/optimizer/Optimizer.cpp

```diff
@@ -20,6 +21,8 @@ using nebula::graph::QueryContext;
 using nebula::graph::Select;
 using nebula::graph::SingleDependencyNode;
 
+DEFINE_bool(enable_optimizer_property_pruner_rule, true, "");
```

Comment:
I think you should reuse `enable_optimizer`

Reply:
How could `enable_optimizer` be able to control whether to enable a single rule?

Reply:
Now we don't provide a way to control single optimization. And in my mind, `enable_optimizer` must control all optimization include this of course.